app/controllers/contact_lyft_rewards_controller.rb

```diff
@@ -41,6 +41,7 @@ def create
         state: 'delivered',
         token: token
       )
+      head :ok
```

Comment:
should this be just json_response

Reply:
hmm I tried this and it seems `json_response()` requires an object. I could do `json_response(nil)`, but that would essentially be the same thing as `head :ok` except we return a `null` in the body instead of an empty body. My thought is that we should leave the body empty since the front-end doesnt require any data, just the response code.

Reply:
Actually we can do `json_response({})`, that gives some flexibility if we want to update the response payload later.